en/config/actuators.md

```diff
@@ -1,94 +1,251 @@
+
+![Reversible](../../assets/config/actuators/qgc_geometry_reversible.png)
+
+Note that you will need to also ensure that the ESC associated with bidirectional motors is configured appropriately (e.g. 3D mode enabled for DShot ESCs, which can be achieved via [DShot commands](../peripherals/dshot.md#commands)).
+
+
+<!-- For example, the roll scale, pitch scale, slew rate and trim can be set (where appropriate) for a control surface, but the positions of these surfaces to each other and to motors cannot be specified. -->
+
+
+### Control Surfaces Geometry
+
+### Control Surface Direction (Conceptual)
+
+![Control Surface Setup Example](../../assets/config/actuators/control_surfaces_geometry.png)
+
+> Example Control Surface Geometry setup for a Tiltrotor VTOL
 
 :::note
-If a servo does not move in the expected direction set in the geometry, you can reverse it using a checkbox on the Actuator Output.
+It is extremely important that we decouple the concept of control surface, servo and motor tilt directions in control allocation from the actual servo output / connections in the physical vehicle.
+
+That means that the Control Allocation Control Surface & Servo settings are physical vehicle independent. Therefore when we think about the surfaces moving, we imagine the ideal vehicle as shown below that moves actuators in the given direction!
 :::
 
+![Control Surface Deflections](../../assets/config/actuators/plane_control_surface_convention.png)
+
+Control surfaces use the following deflection direction convention:
+
+#### Horizontal Control Surface (e.g. Aileron)
+
+> Upwards movement equals Positive deflection
+
+'Pitch UP' control input should command positive deflection to both ailerons
 
-### Motor Tilt Servos
+* Example : To pitch in positive direction (pitching skywards), 'Both Ailerons (Right & Left)' need a positive deflection.
+
+'Roll RIGHT' control input should command positive deflection to right aileron and negative deflection to left aileron
+
+* Example : To roll in positive direction (banking to the right), the 'Right Aileron' needs a positive deflection.
+
+#### Vertical Control Surface (e.g. Rudder)
+
+> Rightwards movement equals Positive Deflection
+
+'Yaw RIGHT' control input should command positive deflection.
+
+* Example : To yaw in positive direction (clockwise rotation), conventional rudder airplane needs a positive deflection.
+
+#### Mixed Control Surface (e.g. V-Tail)
+
+> Upwards movement equals Positive deflection
+
+'Pitch UP' control input should command positive deflection to both V-tails
+
+* Example : To pitch in positive direction (pitching upwards),  'Both V-tail surfaces' need a positive deflection.
+
+> Rightwards movement equals Positive Deflection
+'Yaw RIGHT' control input should command positive deflection to both V-tails.
+
+* Example : Having a V or A shaped tail in the back, positive deflection in Rightwards direction will Yaw the vehicle to the right.
+
+### Motor Tilt Servo Geometry
```

Comment:
OK, so here I updated your Motor tilt bit into its own "geometry" section. It cross links back to the tilt motor section.

@bkueng Arguably, it might be good for this section in the geometry to appear just below the motor configuration, since they are closely linked.

This section not quite complete. In particular, I don't understand the reference frame, and how the vehicles tilt.